File: libraries/base_list/src/main/java/com/tokopedia/abstraction/base/view/adapter/viewholders/EmptyResultViewHolder.java
package com.tokopedia.abstraction.base.view.adapter.viewholders;

import android.view.View;

import androidx.annotation.LayoutRes;

import com.tokopedia.abstraction.base.view.adapter.model.EmptyResultViewModel;
import com.tokopedia.baselist.R;


/**
 * @author kulomady on 1/24/17.
 */

public class EmptyResultViewHolder extends BaseEmptyViewHolder<EmptyResultViewModel> {

    @LayoutRes
    public static final int LAYOUT = R.layout.item_empty_list;

    public EmptyResultViewHolder(View itemView) {
        super(itemView);
    }

    public EmptyResultViewHolder(View itemView, Callback callback) {
        super(itemView, callback);
    }
}

File: features/discovery/similarsearch/src/main/java/com/tokopedia/similarsearch/EmptyResultViewHolder.kt
package com.tokopedia.similarsearch

import android.view.View

internal class EmptyResultViewHolder(itemView: View): BaseViewHolder<EmptyResultViewModel>(itemView) {

    companion object {
        val LAYOUT = R.layout.similar_search_empty_result_layout
    }

    override fun bind(item: EmptyResultViewModel) {

    }
}
